pkg/core/fetch_test.go

```diff
@@ -220,9 +220,7 @@ func Test_Fetch(t *testing.T) {
 	}
 
 	pManager, err := plugin.NewManager(registry.NewRegistryHub(firebase.CloudQueryRegistryURL))
-	if !assert.NoError(t, err) {
-		t.FailNow()
-	}
+	require.NoError(t, err)
```

Comment:
This should fail immediately on error (like the previous code), but with the error details